README.md

```diff
@@ -9,47 +9,25 @@
-
-| Component                                                                       | Description                           | Version                                                                                                          | Lin | Win | Mac |
-| ------------------------------------------------------------------------------- | ------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | --- | --- | --- |
-| [**create-tauri-app**](https://github.com/tauri-apps/create-tauri-app)          | Get started with your first Tauri app | [![](https://img.shields.io/npm/v/create-tauri-app.svg)](https://www.npmjs.com/package/create-tauri-app)         | ✅  | ✅  | ✅  |
-| [**vue-cli-plugin-tauri**](https://github.com/tauri-apps/vue-cli-plugin-tauri/) | Vue CLI plugin for Tauri              | [![](https://img.shields.io/npm/v/vue-cli-plugin-tauri.svg)](https://www.npmjs.com/package/vue-cli-plugin-tauri) | ✅  | ✅  | ✅  |
-
 ## Introduction
 
 Tauri is a framework for building tiny, blazingly fast binaries for all major desktop platforms. Developers can integrate any front-end framework that compiles to HTML, JS and CSS for building their user interface. The backend of the application is a rust-sourced binary with an API that the front-end can interact with.
 
-The user interface in Tauri apps currently leverages [`tao`](https://docs.rs/tao) as a window handling library on macOS and Windows, and [`gtk`](https://gtk-rs.org/docs/gtk/) on Linux via the **Tauri-team** incubated and maintained [WRY](https://github.com/tauri-apps/wry), which creates a unified interface to the system webview (and other goodies like Menu and Taskbar), leveraging WebKit on macOS, WebView2 on Windows and WebKitGTK on Linux.
+The user interface in Tauri apps currently leverages [`tao`](https://docs.rs/tao) as a window handling library on macOS and Windows, and [`gtk`](https://gtk-rs.org/docs/gtk/) on Linux via the **Tauri-team** incubated and maintained [WRY](https://github.com/tauri-apps/wry), which creates a unified interface to the system webview (and other goodies like Menu and Taskbar), leveraging WKWebView on macOS & iOS, WebView2 on Windows, WebKitGTK on Linux and Android System WebView on Android.
 
 To learn more about the details of how all of these pieces fit together, please consult this [ARCHITECTURE.md](https://github.com/tauri-apps/tauri/blob/dev/ARCHITECTURE.md) document.
 
-## Get Started
+## Features
 
-If you are interested in making a tauri app, please visit the [documentation website](https://tauri.app). This README is directed towards those who are interested in contributing to the core library. But if you just want a quick overview about where `tauri` is at in its development, here's a quick burndown:
+The list of Tauri's features includes, but is not limited to:
+
+- Built-in app bundler to create app bundles in formats like `.app`, `.deb`, and `.msi`.
```

Comment:
```suggestion
- Built-in app bundler to create app bundles in formats like `.app`, `.dmg`, `.deb`, `.appimage` and Windows installers like `.exe` (via NSIS) and `.msi` (via WiX).
```

Reply:
i guess you removed AppImage intentionally :joy:

Reply:
i intentionally didn't list all formats and changed the wording to "like ..." to make it clearer that it's only a subset. I figured the list would get quite long if we add .pkg .rpm .snap flatpak/flathub and maybe mobile formats.

Reply:
but ig if we leave the wording as is it's still fine to re-add the v1 formats and not add the new formats (we'll forget anyway)

Reply:
cool it's ok the way it is